### DISCUSSION TITLE
Carthage Preview 3?

### DISCUSSION THREAD

**Discussion by QuintillusCFC** [C7-Game/Prototype#364] [Polls]
Right now we have 24 items in done-but-not-released, versus 47 released in previews 1 and 2.  We also have 4 waiting for review, and 34 ready or in progress, so close to half of what's left is done.

Mathematically, it seems like it may be time for a preview 3.

We also have made some notable improvements, such as adding tile visibility for humans (and thus exploration), accounting for roads in movement costs, and including the first community-created art asset.  As well as addressing 8 bugs (with 2 more in review status).

So, maybe it's time for a Carthage Preview 3?  Or is there something we should wait for?  Or, now that all of the initially planned Carthage milestones are at least somewhat available, should we (now or soon) release a full Carthage and move the remainder to Dutch?

**Comment by QuintillusCFC** [comment]
I'm favoring option 1 (release Preview 3), but can see arguments for all of them.

Preview 3 continues the current pattern.

We could wait for the final Carthage items, but IMO it's already been long enough, and we have enough.  Still, that would lead to a bigger changeset.

I think we put too many items in Carthage overall, and should have had a smaller scope, so I have some sympathy for moving the remainder to Dutch.  But while I think the next big items should come from Dutch (e.g. buildings, technologies), there are enough small items in the "To do" column that while I don't think we should necessarily do all of them before starting on the next big items, we should do some of them, especially keeping the bug count down.

On bugs, despite the 8 fixed, we're only at a net decrease of 1 bug since Preview 2 since more of them keep being identified.  Part of this is doing a better job of documenting them, part of it is that some "bugs" are incorrect because we haven't added the finer points of some features yet.  But nonetheless, I was hoping we could reduce our count from 20 to 10, and even if we finish the two in review, we'll still be at 17.

**Comment by WildWeazel** [comment]
I continue to favor frequent releases, even if they're small increments. I do agree we probably assigned too much to Carthage, but I'd really prefer to sort out our architecture and reorganization before moving on. *glares in mirror*

**Reply by QuintillusCFC** [reply]
All right, I'll take that as the go-ahead, and have thus built and released Carthage Preview 3 (https://forums.civfanatics.com/threads/release-carthage-preview-3.679663/)
